docs/introduction/Base-App.rst

```diff
@@ -403,6 +403,16 @@ Must be an instance of a class implementing the com.horizen.storage.Storage inte
     	.annotatedWith(Names.named("StateUtxoMerkleTreeStorage"))
     	.toInstance(..);
 
+-  BackupStorage
+Storage containing the non coin-boxes saved during the backup procedure.
```

Comment:
Let's add the info that in case we don't want to have any restore logic, then just keep this as an empty storage.

And also let's add the link to the `8.10. Backup and restore procedure` section

Reply:
Done